dwave_networkx/algorithms/elimination_ordering.py

```diff
@@ -185,8 +188,6 @@ def min_fill_heuristic(G):
     This example computes an upper bound for the treewidth of the :math:`K_4`
     complete graph.
 
-    >>> import dwave_networkx as dnx
-    >>> import networkx as nx
```

Comment:
Will users know what "nx" and "dnx" represent without the import? Shouldn't the following lines be updated to something like:
```
 >>> K_4 = networkx.complete_graph(4)
 >>> tw, order = dwave_networkx.min_fill_heuristic(K_4)
```
Same for following examples

Reply:
I think in this library it is safe to assume that they know. NetworkX follows this convention (to pick a random example https://networkx.github.io/documentation/stable/reference/algorithms/generated/networkx.algorithms.bridges.bridges.html#networkx.algorithms.bridges.bridges)